src/Provider.tsx

```diff
@@ -81,9 +79,9 @@ export class OptimizelyProvider extends React.Component<OptimizelyProviderProps,
       finalUser = DefaultUser;
```

Comment:
Shouldn't it be like this since we have keep userAttrobutes?
```
finalUser = {
   id: null,
  attributes: userAttributes || {},
```

Reply:
Updating...